packages/framework/src/Model/Product/Search/ProductElasticsearchRepository.php

```diff
@@ -7,27 +7,26 @@
 use Doctrine\ORM\QueryBuilder;
 use Elasticsearch\Client;
 use Shopsys\FrameworkBundle\Component\Elasticsearch\IndexDefinitionLoader;
+use Shopsys\FrameworkBundle\Component\LocalCache\LocalCacheFacade;
 use Shopsys\FrameworkBundle\Model\Product\Elasticsearch\ProductIndex;
-use Symfony\Contracts\Service\ResetInterface;
 
-class ProductElasticsearchRepository implements ResetInterface
+class ProductElasticsearchRepository
 {
-    /**
-     * @var int[][][]
-     */
-    protected array $foundProductIdsCache = [];
+    protected const string FOUND_PRODUCT_IDS_CACHE_NAMESPACE = 'foundProductIdsCache';
```

Comment:
You do not use word `Cache` in previous strings, so remove it here too.

Reply:
i dont understand what you meen